spec/controllers/jobs_controller_spec.rb

```diff
@@ -92,4 +92,18 @@
       expect(Delayed::Job.find(@running.id)).to be
     end
   end
+
+  describe "POST retry_queued" do
+    before do
+      @not_running = Delayed::Job.create(run_at: Time.zone.now - 1.hour)
+      @not_running.update_attribute(:attempts, 1)
+      sign_in users(:jane)
+    end
+
+    it "run the queued job" do
+      expect(Delayed::Job.last.run_at.to_s).not_to eq(Time.zone.now.to_s)
+      Delayed::Job.awaiting_retry.update_all(run_at: Time.zone.now)
```

Comment:
I did try that but I was getting 

```
  3) JobsController POST retry_queued run the queued job
     Failure/Error: post :retry_queued
     ArgumentError:
       wrong number of arguments (given 2, expected 1)
```
Can I just do `post :retry_queued` without actually posting anything?